packages/remotemetrics/events.go

```diff
@@ -70,6 +70,7 @@ type MessageScheduledMetrics struct {
 	MessageID                string    `json:"messageID" bson:"messageID"`
 	TransactionID            string    `json:"transactionID,omitempty" bson:"transactionID"`
 	IssuedTimestamp          time.Time `json:"issuedTimestamp" bson:"issuedTimestamp"`
+	ReceivedTimestamp        time.Time `json:"receivedTimestamp" bson:"receivedTimestamp"`
```

Comment:
What about difference between received and issued timestamp? We cannot easily calculate it in Elasticsearch, so this would need to be added if we want to measure the delay?

Reply:
I can add comments if you want

Issued time - encoded into the message. The time set by the node that created it.
ReceivedTimestamp - the time that your node received a meesage

your question made me remember that there is a little bug that should be fixed
So I'll commit something in a second

Reply:
I think processing timestamp and delta shceduled have confusing names currently

you have recommendations?

Reply:
I meant that we could have another field here, that would be the difference `ReceivedTimestamp - IssuedTimestamp`.

Reply:
we already have it `ProcessingTime`

Reply:
I think the names are bad, see my other comment
but not sure what the new names should be @piotrm50